OpenRA.Mods.Common/EditorBrushes/EditorDefaultBrush.cs

```diff
@@ -51,6 +52,15 @@ public EditorDefaultBrush(EditorViewportControllerWidget editorWidget, WorldRend
 				.ToDictionary(r => r.Info.ResourceType, r => r);
 		}
 
+		long CalculateActorSelectionPriority(EditorActorPreview actor)
+		{
+			var center = new int2(actor.Bounds.X, actor.Bounds.Y);
+			long priority = (center - worldPixel).Length;
+			priority <<= 32;
```

Comment:
I agree that this is standard, however a lot of people are scared of low level bit-wrangling.  I think @penev92's suggestion of adding a short comment is good.